src/interceptor.ts

```diff
@@ -175,6 +181,7 @@ export const aws4Interceptor = <D = any>({
       region: options?.region,
       service: options?.service,
       signQuery: options?.signQuery,
+      addContentSha: options?.addContentSha,
```

Comment:
@dblock this is throwing a type error, the underling `aws4` library knows nothing of this option, so I guess we don't need to pass it through here?